soccer/gameplay/tactics/positions/submissive_defender.py

```diff
@@ -151,6 +199,29 @@ def execute_marking(self):
     def on_exit_marking(self):
         self.remove_subbehavior('move')
 
+    def on_enter_clearing(self):
+        print("CLEARING BALL")
+        #copied from submissivegoalie
+        kick = skills.pivot_kick.PivotKick()
+
+        kick.aim_params['error_threshold'] = 1.0
+        kick.aim_params['max_steady_ang_vel'] = 12
+
+        # chip
+        kick.chip_power = 1.0
+        kick.use_chipper = True
+
+        kick.target = robocup.Segment(
+            robocup.Point(-constants.Field.Width / 2, constants.Field.Length),
+            robocup.Point(constants.Field.Width / 2, constants.Field.Length))
```

Comment:
It might be good to make the target segment a little narrower so that it tends to kick the ball more length-wise down the field than sideways - this'll help make it more likely that the ball stays in play rather than getting kicked out towards the side.